charts/oes/templates/deployments/oes-rabbitmq-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: oes
+    component: rabbitmq
+  name: rabbitmq
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: oes
+      component: rabbitmq
+  template:
+    metadata:
+      labels:
+        app: oes
+        component: rabbitmq
+    spec:
+      containers:
+      - image: {{ .Values.rabbitmq.image.registry }}:{{ .Values.rabbitmq.image.repository }}
```

Comment:
We should probably pull rabbitmq image to our registry and use our quay.io/opsmxpublic registry here